packages/logs/src/boot/logs.entry.ts

```diff
@@ -53,7 +53,7 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
     logger,
 
     init: monitor((userConfiguration: LogsUserConfiguration) => {
-      if (!checkIsNotLocalFile() || !canInitLogs(userConfiguration)) {
+      if (!(userConfiguration.allowLocalFile || checkIsNotLocalFile()) || !canInitLogs(userConfiguration)) {
```

Comment:
There is a failing test on the case where init is called without configuration.
Checking `canInitLogs` first should solve that